### PR TITLE
Fix up a pointer to the docker v1 spec

### PR DESCRIPTION
**Pull request by duglin**
I'm not sure this is correct, but the ref to "manifest-v2-1.md" was returning a 404 and I think it should point to the docker image v1 spec - at the same time I also renamed "serialization.md" to "docker-image-v1.0.0" since that's more descriptive - assuming that's the right name for it.

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: manifest.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This document outlines the format of of the V2 image manifest, schema version 2.
 The original (and provisional) image manifest for V2 (schema 1), was introduced
 in the Docker daemon in the [v1.3.0
 release](https://github.com/docker/docker/commit/9f482a66ab37ec396ac61ed0c00d59122ac07453)
-and is specified in the [schema 1 manifest definition](./manifest-v2-1.md)
+and is specified in the [schema 1 manifest definition](./docker-image-v1.0.0.md)
```
Comment (vbatts) [inline manifest.md]: ah, this should just be `manifest.md`

Comment (duglin) [inline manifest.md]: So it should point to itself?

Comment (vbatts) [inline manifest.md]: ah. this whole sentence could be axed.

Comment (philips) [inline manifest.md]: +1 on axing it

**Comment by philips** [comment]
@duglin Ping. Otherwise I will fix this myself today.

**Comment by duglin** [comment]
I think it's already been done, no?

**Comment by philips** [comment]
@duglin wait, what? You renamed serialization to docker v1? I don't think that was the goal. I thought you were just going to remove that sentence.  https://github.com/opencontainers/image-spec/pull/19#discussion_r59558935

**Comment by duglin** [comment]
yea I was focusing on that sentence too but it appears to have vanished from master - unless I'm looking at the wrong spot

**Comment by philips** [comment]
ah, in that case I will close this out.

**Comment by duglin** [comment]
ok

**Pull request closed without merging**